docs/resources/cts_notification.md

```diff
@@ -0,0 +1,103 @@
+---
+subcategory: "Cloud Trace Service (CTS)"
+---
+
+# huaweicloud_cts_notification
+
+Manages CTS key event notification resource within HuaweiCloud.
+
+## Example Usage
+
+### Complete Notification
+
+```hcl
+variable "topic_urn" {}
+
+resource "huaweicloud_cts_notification" "notify" {
+  name           = "keyOperate_test"
+  operation_type = "complete"
+  smn_topic      = var.topic_urn
+}
+```
+
+### Customized Notification
+
+```hcl
+variable "topic_urn" {}
+
+resource "huaweicloud_cts_notification" "notify" {
+  name           = "keyOperate_test"
+  operation_type = "customized"
+  smn_topic      = var.topic_urn
+
+  operations {
+    service     = "ECS"
+    resource    = "ecs"
+    trace_names = ["createServer", "deleteServer"]
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to manage the CTS notification resource.
+  If omitted, the provider-level region will be used. Changing this creates a new resource.
+
+* `name` - (Required, String) Specifies the notification name. The value contains a maximum of 64 characters,
+  and only letters, digits, underscores (_), and Chinese characters are allowed.
+
+* `operation_type` - (Required, String) Specifies the operation type, possible options include **complete** and
+  **customized**.
+
+* `smn_topic` - (Optional, String) Specifies the URN of a topic.
+
+* `operations` - (Optional, List) Specifies an array of operations that will trigger notifications.
+  For details, see [Supported Services and Operations](https://support.huaweicloud.com/intl/en-us/usermanual-cts/cts_03_0022.html).
+  The [object](#notification_operations_object) structure is documented below.
+
+* `notification_users` - (Optional, List) Specifies an array of users. SMN will notify subscribers of key operations
+  initiated by the specified users. The [object](#notification_users_object) structure is documented below.
```

Comment:
pls add default: trace all users.   s/notification_users/trace_users

Reply:
rename it to **operation_users**